pages/docs/community/styleguide/content-buckets.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Content Buckets
+Description: This style guide explains how and why we use content buckets in documentation.
+weight: 30
+---
+
+# Content Buckets
+To structure our documentation, we adopt a systematic approach from the Diátaxis framework. We ensure our docs are presented in a manner that is easy to understand by organizing and classifying them into content buckets.
+
+![Diátaxis framework](https://www.asyncapi.com/img/posts/changes-coming-docs/diataxis.webp)
```

Comment:
```suggestion

```

I love adding this visual, but we shouldn't link to blog post images inside a docs page. Additionally, we should avoid adding assets to docs because they require extra maintenance, and we have yet to determine guidelines for where to store docs assets and how to ensure they stay up to date.

Since the info in this asset can easily be made into a mermaid diagram, please delete this image and instead add a mermaid diagram.